src/main/java/frc/robot/subsystems/intake/commands/TakeIn.java

```diff
@@ -0,0 +1,30 @@
+package frc.robot.subsystems.intake.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants;
+import frc.robot.subsystems.intake.Intake;
+
+public class TakeIn extends CommandBase {
+    private Intake intake;
+    public TakeIn(Intake i) {
+        intake = i;
+        addRequirements(intake);
+    }
+
+    @Override
+    public void initialize() {
+        if(!intake.isOpen())
+            intake.togglePiston();
+        intake.setVelocity(Constants.Intake.VELOCITY);

```

Comment:
A command should do only 1 thing.
This function does 2 things:
1) open the retractor.
2) set the velocity for the wheels

Reply:
Also, don't you have a `setPistonMode` (or something similar) function, if not, please add it

Reply:
if you want to use this command to power the wheels, you need the `execute` function

Reply:
also, split it into a command group